src/WorkBC.Tests/Fixtures/WantedXmlJobs/6178210923.xml

```diff
@@ -0,0 +1,39 @@
+﻿<job id="6178210923" hash="2204234286" refnumber="" isstaffing="false" isanonymous="false" isthirdparty="false"
+     isinappropriate="false" isbulk="false" isaggregator="false" isfree="false" isclassifiedoccupation="true"
+     isclassifiedindustry="true" iscurrent="true">
+	<dates firstseen="2019-10-03" posted="2019-10-03" refreshed="2019-10-03" />
+	<title value="Digital Marketing Specialist - Bilingual" titleid="698109398" cleantitleid="12100081"
+		   semicleantitleid="37141148" />
+	<description
+	  value="Bilingual (French/English) Digital Marketing Specialist WE NEED Absolute Results is looking for a Bilingual Digital Marketing Specialist to join our fast-growing team in Surrey, BC. Reporting to the Senior Digital Marketing Lead, you will manage..." />
+	<occupation>
+		<noc code="1111" label="Accountant" revision="2011" />
```

Comment:
Please make sure to include NOC 2016 codes that are present in various positions of the NocCode2016 field:
- single entry
- multiple entries, first position
- multiple entries, middle position
- multiple entries, last position

Reply:
Added more unit tests in SearchTests with multiple 2016 Noc Codes tagged to 2021 Noc Codes. These unit tests are currently designed to test for Jobs procured from Federal and Wanted sources.